docs/using-capture-growth-charts.md

```diff
@@ -4,58 +4,156 @@
+- `<Url of instance>/api/apps/capture-growth-chart/plugin.html` 
+
+Remember to alter `<Url of instance>` with the actual url of you instance.
+<br />
+
+# Configuration { #configuration }
+## Maintenance { #maintenance }
+
+### Dataelement
+Create data elements for **Weight**, **Height** and **Head circumference**.
+Weight can be in either `gram` or `kg`, but height and head circumference should be in `cm`.
+ <br />
+
+### Program
+#### Tracked entity attribute
+Tracked entity attribues needed for the Growth chart plugin is `Date of birth` and `gender`. However, `First Name` and `Last Name` are also needed if you want the name to be printed when using the print function. <br /> <br />
+                    
+#### Tracked entity type
+Tracked entity type for person needs the same attributes as the tracked entity attributes. <br /> <br />
+
+
+#### Program     
+##### Attributes
+Select you preffered program for storing the growth variables and displaying the Growth Chart.
+The program should have the following attributes:
+- `First name`
+- `Last name`
+- `Date of birth`
+- `Gender`
+ <br />
+
+##### Program stages
+Select stage where growth variables currently are or will be stored.
+The program stage should have the following data elements:
+- `Weight` (g or kg)
+- `Height` (cm)
+- `Head circumference` (cm)
+<br /> <br />               
+                         
+## Datastore Manangement { #datastore_management }
+### Capture
+In namespace `capture`, enter the file with key `enrollmentOverviewLayout`
+Add new section for the growth chart under `leftColumn`. You can choose where on the left column to place it. Add the following code, but remember to change out `<Url of instance>` with the url of your instance.
+```json
+{
+    "source": "<Url of instance>/api/apps/capture-growth-chart/plugin.html",
+    "type": "plugin"
+}
+```
+<br />
+
+### Capture-growth-chart
+#### Config      
+Create new namespace `capture-growth-chart` with key `config`
+The growth chart plugin needs this config to work. All Id's should be changed, and will be specific for each implementation. The `femaleOptionCode` and `maleOptionCode` should map to the option codes used for gender. The structure of the config has to be the same as the one in the example below;
+```json
+{
+    "metadata": {
+        "attributes": {
+            "dateOfBirth": "AMl8BkN8Lyq",
+            "gender": "tyNlJWNnEbs",
+            "femaleOptionCode": "CGC_Female",
+            "maleOptionCode": "CGC_Male"
+        },
+        "dataElements": {
+            "headCircumference": "GfchA70xtmP",
+            "height": "wWCSulSdUgd",
+            "weight": "yZwKJdYXTZF"
+        },
+        "program": {
+            "programStageId": "h3gT08Et4sC"
+        }
+    },
+    "settings": {
+        "customReferences": false,
+        "defaultStandard": "ID",
+        "weightInGrams": false
+    }
+}
+```    
+<br />
+
+#### Custom references
```

Comment:
Remove this part, not ready!

Reply:
Will keep it for now, but mark it as future features